src/backy/sources/ceph/source.py

```diff
@@ -95,12 +90,12 @@ def backup(self, target):
             break
         self.diff(target, parent)
 
-    def diff(self, target, parent):
+    def diff(self, target_: BackyBackend, parent: Revision) -> None:
```

Comment:
From an outside perspective it would be more graceful to use the `target_` variant internally.